api/envoy/api/v2/route/route_components.proto

```diff
@@ -436,7 +436,7 @@ message RouteMatch {
     type.matcher.RegexMatcher safe_regex = 10 [(validate.rules).message = {required: true}];
   }
 
-  // Indicates that prefix/path matching should be case insensitive. The default
+  // Indicates that prefix/path/regex/safe_regex matching should be case insensitive. The default
```

Comment:
The v2 APIs are frozen, so it only makes sense to do this in v3.

Reply:
We still use v2, so are many more, would be nice if it can be accommodated.

Reply:
Sorry I don't think this will make the cut for v2.

Reply:
Okay I'll rollback v2 changes